docs/source/scheduling.rst

```diff
@@ -208,3 +208,9 @@ reusable ``ProcessPoolExecutor`` from loky_:
    from loky import get_reusable_executor
    with dask.config.set(pool=get_reusable_executor(max_workers=4)):
        x.compute()
+
+Other libraries like ipyparallel_ and mpi4py_ also supply
+``concurrent.futures.Executor`` subclasses that could be used as well.
+
+.. _ipyparallel: https://ipyparallel.readthedocs.io/en/latest/api/ipyparallel.html#ipyparallel.Client.executor
+.. _mpi4py: https://mpi4py.readthedocs.io/en/stable/mpi4py.futures.html
```

Comment:
We've been slowly making progress toward the next release, but for the time being I'd suggest people (especially HPC people) to always refer to the latest docs. While the stable branch (for v3.0.3) does have `mpi4py.futures` included, it missed a lot of new features we added in the past ~2 years.
```suggestion
.. _mpi4py: https://mpi4py.readthedocs.io/en/latest/mpi4py.futures.html
```

Reply:
Makes sense. Updated :)